exonum/src/blockchain/service.rs

```diff
@@ -410,6 +410,34 @@ impl SharedNodeState {
         }
     }
 
+
+    /// Returns the majority count from the current "State"
+    /// of a blockchain node.
+    pub fn majority_count(&self) -> usize {
+        self.state
+            .read()
+            .expect("Expected read lock.")
+            .majority_count
+    }
+    /// Updates internal majority count, from `State` of a blockchain node.
+    pub fn update_majority_count(&self, state: &State) {
```

Comment:
Is this function needed? You don't use it anywhere.

Reply:
it uses in node/basic.rs